site/src/pages/DeploySettingsPage/optionValue.ts

```diff
@@ -41,7 +41,7 @@ export function optionValue(
       const experimentMap: Record<string, boolean> | undefined =
         additionalValues?.reduce(
           (acc, v) => {
-            return { ...acc, [v]: option.value.includes("*") ? true : false };
+            return { ...acc, [v]: option.value.includes("*") };
           },
           {} as Record<string, boolean>,
         );
```

Comment:
Didn't realize we had code like this. The `includes` change is good, but I think there's room to go two steps further, depending on what we want to do:

The types can be cleaned up:
```ts
const experimentMap = inputs?.reduce<Record<string, boolean>>((acc, v) => {
  return { ...acc, [v]: option.value.includes('*') };
}, {});
```
And then the spread operator can be replaced with a "safe mutation":
```ts
const experimentMap = inputs?.reduce<Record<string, boolean>>((acc, v) => {
  acc[v] = option.value.includes('*');
  return acc;
}, {});
```
The main issue with the spread syntax is that while it's clean looking, when it's inside a loop like this, the logic is creating a brand new object on every single iteration, and then only keeping the final one.

Typically mutations in React are bad, but in the grand scheme of things, the entire `experimentMap` expression is still pure and deterministic. It's just a pure result via impure means